pixl_core/src/core/project_config.py

```diff
@@ -87,6 +87,7 @@ class PixlConfig(BaseModel):
     destination: _Destination
 
     @field_validator("tag_operation_files")
+    @classmethod
```

Comment:
```suggestion
```

Not strictly needed, as we define it in `pyproject.toml` under `classmethod-decorators`

Reply:
I put it in because the pydantic docs recommend it, but eh

Reply:
You're right, I was going off from an older version of pydantic (which we were using before). I got rid of the `pyproject.toml` line as we don't need it anymore and added the `@classmethod` to the other field validator as well.